src/compute-client/src/service.rs

```diff
@@ -243,6 +253,10 @@ where
                     .collect()
             }
             command => {
+                if let ComputeCommand::UpdateConfiguration(config) = &command {
+                    self.max_result_size = config.max_result_size;
+                }
+
```

Comment:
I think this is best pulled before the `match command` because it logically is a different step. The current `match command` only shards the command into partitions, but doesn't interpret the commands in any way. What you're adding is an interpretation, so it seems better to do that separately.

Reply:
Should the `ComputeCommand::UpdateConfiguration` continue to get forwarded on to the first shard?

Reply:
Moved `UpdateConfiguration` to its own branch, and kept the forwarding of the command!